src/types/models/listing.ts

```diff
@@ -132,6 +132,7 @@ export interface Listing
   interior: string
   kiloWatts: number
   leasingMonthlyRate: number
+  loanMonthlyRate: number
```

Comment:
Shouldn't this be optional?

Reply:
BE sends both `leasingMonthlyRate` and `loanMonthlyRate`. if `leasingMonthlyRate` exists -> the user is ineligible for a loan and `loanMonthlyRate` is null. And the other way around. Confirmed with BE